### DISCUSSION TITLE
What is the name of Greece's largest island?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#618] [Galaxy Trivia] [answered]
*known for beaches, archaeology sites and rugged landscapes*

**Comment by peterbaccaro** [comment]
The largest Greek island by both area and population is **Crete**, located at the southern edge of the Aegean Sea.

**Comment by thuongtruong1009** [comment, marked as answer]
The largest island in Greece is called Crete.
